controllers/front/listing/CategoryController.php

```diff
@@ -80,7 +80,7 @@ public function init()
 
         parent::init();
 
-        if (!Validate::isLoadedObject($this->category) || !$this->category->active) {
+        if (!Validate::isLoadedObject($this->category) || !$this->category->active || $this->category->existsInShop($this->context->shop->id)) {
```

Comment:
```suggestion
        if (!Validate::isLoadedObject($this->category) || !$this->category->active || !$this->category->existsInShop($this->context->shop->id)) {
```

I think you wanted to check whether it doesn't exist. Right?

Reply:
also, I think you could send this fix against 8.0.x branch

I'm pretty sure @Hlavtox fixed it some time ago, not sure what happened that it's still a problem 🤔 any hints Daniel?

Reply:
Note, breadcrumbs on concerned 404 page also need to be fixed.
The full path of the other shop category is leaked

Reply:
@kpodemski I think it was customer groups that we fixed or improved, there was also a wrong var used. But nothing with multistore. :-)

But from looking at it, it's a legit fix.